src/utils/XR.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * @namespace xr
+ * @description Useful namespace for helping with xr utility functions.
+ * this is set within the MRApp to access various WebXR API features
+ */
+let xr = {};
```

Comment:
Can we use `XR` instead of `xr` ?

Reply:
xr is consistent with the web standard `navigator.xr` or `renderer.xr`